meilisearch-release-changelog/v1.3.0.md

```diff
@@ -0,0 +1,241 @@
+# v1.3.0 release changelogs
+
+Meilisearch v1.3.0 introduces Meilisearch as a vector store, it introduces visible ranking score details and defining a field to search on at search time.  It also now includes the ability to search within facet values and facet values that are sortable.
+
+<!-- The following lines should ONLY be put in the official release changelogs -->
+🧰 All official Meilisearch integrations (including SDKs, clients, and other tools) are compatible with this Meilisearch release. Integration deployment happens between 4 to 48 hours after a new version becomes available.
+
+Some SDKs might not include all new features—consult the project repository for detailed information. Is a feature you need missing from your chosen SDK? Create an issue letting us know you need it, or, for open-source karma points, open a PR implementing it (we'll love you for that ❤️).
+
+# New features and improvements 🔥
+
+## Vector Store - Experimental
+The release of v1.3.0 enables Meilisearch to serve as a vector store. This means Meilisearch now accepts vectors in documents, stores them, and lets users retrieve the nearest documents using the search endpoint.
+
+This experimental feature can be enabled at runtime using v1.3.0's new `/experimental-features` endpoint.
```

Comment:
Which means that could can try it directly on the Cloud, DAY ONE ☝️

Reply:
I added in a link there too with our OSS tracker on it 👍🏻